modules/pytket-braket/pytket/extensions/braket/backends/braket.py

```diff
@@ -203,20 +203,21 @@ def _obs_from_qpo(operator: QubitPauliOperator, n_qubits: int) -> Observable:
 
 def _get_result(
     completed_task: Union[AwsQuantumTask, LocalQuantumTask],
-    n_qubits: int,
+    target_qubits: str,
     want_state: bool,
     want_dm: bool,
     ppcirc: Optional[Circuit] = None,
 ) -> Dict[str, BackendResult]:
     result = completed_task.result()
     kwargs = {}
+    target_qubits = json.loads(target_qubits)
```

Comment:
Please use a different variable for this.

Reply:
Done